crates/indexer/src/processors/token_processor.rs

```diff
@@ -486,11 +515,17 @@ impl TransactionProcessor for TokenTransactionProcessor {
         let table_handle_to_owner =
             TableMetadataForToken::get_table_handle_to_owner_from_transactions(&transactions);
 
+        // get aptos_coin info for supply tracking
+        // TODO: This only needs to be fetched once. Need to persist somehow
+        let maybe_aptos_coin_info =
+            &CoinInfoQuery::get_by_coin_type(APTOS_COIN_TYPE.to_string(), &mut conn).unwrap();
```

Comment:
Is this permanent? I think you should be able to reuse the coin stuff no?

Reply:
It is just copied from the way coin_processor computes the CoinActivity.

https://github.com/aptos-labs/aptos-core/blob/12491736878d172d730cfe4cda9e8e50239e6328/crates/indexer/src/processors/coin_processor.rs#L256-L259

https://github.com/aptos-labs/aptos-core/blob/12491736878d172d730cfe4cda9e8e50239e6328/crates/indexer/src/processors/coin_processor.rs#L269-L275